node/src/components/chainspec_loader/config.rs

```diff
@@ -26,6 +27,14 @@ const DEFAULT_UPGRADE_INSTALLER_PATH: &str = "upgrade_install.wasm";
 const DEFAULT_VALIDATOR_SLOTS: u32 = 5;
 const DEFAULT_AUCTION_DELAY: u64 = 3;
 const DEFAULT_LOCKED_FUNDS_PERIOD: EraId = 15;
+/// Round seigniorage rate represented as a fractional number
+///
+/// Annual issuance: 2%
+/// Minimum round exponent: 14
+/// Ticks per year: 31536000000
+///
+/// (1+0.02)^((2^14)/31536000000)-1 is expressed as a fraction below.
+const DEFAULT_ROUND_SEIGNIORAGE_RATE: Ratio<u64> = Ratio::new_raw(6414, 623437335209);
```

Comment:
Should this have a default value at all?
I don't think we have a default round exponent.

Reply:
I see your point though I need _something_ for `impl Default for Genesis` below. I used the default value as it was hardcoded in EE so far

Reply:
We shouldn't have `Default for Genesis` at all :grimacing: